src/assets/translations/en/main.json

```
@@ -373,6 +373,7 @@
         "vaultCap": "Vault Cap",
         "vaultCapTooltip": "Current maximum capacity for this vault",
         "haltedTitle": "Deposits are temporarily halted.",
```

nitpick rename suggestion
```suggestion
        "haltedDepositTitle": "Deposits are temporarily halted.",
```

---

2c51df55d6607a077137e8b3a8fc3679a67fe845